pkg/broker/util.go

```diff
@@ -196,7 +197,7 @@ func createUIFormItem(pd apb.ParameterDescriptor, paramIndex int) (interface{},
 
 // getType transforms an apb parameter type to a JSON Schema type
 func getType(paramType string) (schema.PrimitiveTypes, error) {
-	switch paramType {
+	switch strings.ToLower(paramType) {
```

Comment:
Can we add a test case for this? `util_test.go:371` should be the test cases to add to?

Reply:
Do you have something in mind that's more than what I already added there?

Reply:
```go
...
{"number", schema.NumberType},
{"NUMBER", schema.NumberType},
{"nil", schema.NullType},
...
```

Add that to line 385ish in `pkg/broker/util_test.go`